### PR TITLE
feat(sigma): Add support for Lacework agent data

### PR DESCRIPTION
**Pull request by hazedav**
Support linux.file_create
Support linux.process_creation

Example usage:
```
❯ $SIGMAC_CMD -t lacework $SIGMA_HOME/rules/linux/process_creation/lnx_local_account.yml                                                    
---
evaluatorId:
queryId: Sigma_Local_System_Accounts_Discovery
queryText: |-
  Sigma_Local_System_Accounts_Discovery {
      source {
          LW_HE_PROCESSES
      }
      filter {
          (EXE_PATH LIKE '%/lastlog' or contains(CMDLINE, '\'x:0:\'') or (EXE_PATH LIKE '%/cat' and contains(CMDLINE, '/etc/passwd') or contains(CMDLINE, '/etc/sudoers')) or EXE_PATH LIKE '%/id' or (EXE_PATH LIKE '%/lsof' and contains(CMDLINE, '-u'))) and diff_minutes(PROCESS_START_TIME, current_timestamp_sec()::timestamp) <= 60
      }
      return distinct {
          RECORD_CREATED_TIME,
          MID,
          PID,
          EXE_PATH,
          CMDLINE,
          CWD,
          ROOT,
          USERNAME,
          PROCESS_START_TIME
      }
  }
---
evaluatorId:
title: Local System Accounts Discovery
enabled: false
policyType: Violation
alertEnabled: false
alertProfile: LW_HE_Files.HE_Process_NewViolation
evalFrequency: Hourly
queryId: Sigma_Local_System_Accounts_Discovery
limit: 1000
severity: low
description: Detects enumeration of local systeam accounts
remediation: Remediation steps are not represented in Sigma rule specification
```

### CODE REVIEW

File: tools/sigma/backends/lacework.py [hunk 1]
```diff
@@ -62,6 +61,73 @@
...
+          - RECORD_CREATED_TIME
+          - MID
+          - PATH
+          - FILE_TYPE
+          - SIZE
+          - FILEDATA_HASH
+          - OWNER_UID
+          - OWNER_USERNAME
+          - FILE_CREATED_TIME
+        alertProfile: LW_HE_Files.HE_File_NewViolation
```
Comment (rachelrice) [inline tools/sigma/backends/lacework.py]: i saw in the laceup-lpp-for-red-team channel Djay shared an updated list of alert profile names, i think this should now be `LW_HE_FILES_DEFAULT_PROFILE.HE_File_NewViolation`

File: tools/sigma/backends/lacework.py [hunk 2]
```diff
@@ -62,6 +61,73 @@
...
+          - RECORD_CREATED_TIME
+          - MID
+          - PID
+          - EXE_PATH
+          - CMDLINE
+          - CWD
+          - ROOT
+          - USERNAME
+          - PROCESS_START_TIME
+        alertProfile: LW_HE_Files.HE_Process_NewViolation
```
Comment (hazedav) [inline tools/sigma/backends/lacework.py]: Looks like I have this wrong as well...

Comment (rachelrice) [inline tools/sigma/backends/lacework.py]: and this perhaps `LW_HE_PROCESSES_DEFAULT_PROFILE.HE_Process_NewViolation`

**Review by rachelrice — APPROVED** [review]
looks good to me, just a query on the alertProfile names used

File: tools/sigma/backends/lacework.py (hunk 2 above)
Comment (hazedav) [reply tools/sigma/backends/lacework.py]: Fixed.

Comment (hazedav) [reply tools/sigma/backends/lacework.py]: Fixed.

File: tools/sigma/backends/lacework.py (hunk 1 above)
Comment (hazedav) [reply tools/sigma/backends/lacework.py]: Fixed.

**Pull request merged**